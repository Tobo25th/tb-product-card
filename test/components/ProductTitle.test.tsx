import React from "react";
import renderer from "react-test-renderer";
import {ProductTitle,ProductCard} from "../../src/components"
import {product1} from "../data/products";


describe('ProductTitle', () => {
    test("should be show the component with the custom title correctly",()=>{
        const wrapper = renderer.create(
            <ProductTitle title="custom product" className="customclass"/>
        )

        
            expect(wrapper.toJSON()).toMatchSnapshot();

    });
    test('it should display the product title', () => {
        const wrapper = renderer.create(
            <ProductCard product={product1} className="customclass">
                {()=>(
                    <ProductTitle/>
                )}
            </ProductCard>
        )

        
            expect(wrapper.toJSON()).toMatchSnapshot();



    });

})