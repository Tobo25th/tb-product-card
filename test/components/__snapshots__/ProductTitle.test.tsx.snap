// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle it should display the product title 1`] = `
<div
  className="productCard customclass"
>
  <span
    className="productDescription undefined"
  >
    Coffee-Mug - Card without image
  </span>
</div>
`;

exports[`ProductTitle should be show the component with the custom title correctly 1`] = `
<span
  className="productDescription customclass"
>
  custom product
</span>
`;
